aten/src/ATen/core/Reduction.h

```diff
@@ -2,7 +2,7 @@
 
 namespace Reduction {
 
-// NB: Keep this in sync with Reduction class in torch/nn/modules/functional.py
+// NB: Keep this in sync with Reduction class in torch/nn/_reduction.py
```

Comment:
Is this a related change? If not, please split it.

Reply:
not related to this but can we just put it here? since it's just a small comment change.